addons/sale_stock/static/src/product_catalog/kanban_record.js

```diff
@@ -1,8 +1,17 @@
 /** @odoo-module */
-import { ProductCatalogKanbanRecord } from "@product/product_catalog/kanban_record";
+import { ProductCatalogKanbanRecord, DefaultProductCatalogKanbanRecord } from "@product/product_catalog/kanban_record";
 import { ProductCatalogSaleOrderLine } from "./sale_order_line/sale_order_line";
 import { patch } from "@web/core/utils/patch";
 
+patch(DefaultProductCatalogKanbanRecord.prototype, {
```

Comment:
And if we pass in the `productCatalogData` the type of the product to know if it is a storable product, then we could just apply the `updateQuantity` patch in that file only for those products type, no? 🤔

Reply:
@xavierbol mmh yeah this should be simpler, check my fix and tell me what you think